Packet++/header/WireGuardLayer.h

```diff
@@ -0,0 +1,254 @@
+#pragma once
+
+#include "Layer.h"
+#include "IpAddress.h"
+#include "MacAddress.h"
+#include <string.h>
```

Comment:
CI is failing because `std::memcmp` used in the tests is not guaranteed to be declared in `string.h` as it is in `cstring`. Either use `cstring` or use the memcmp from the global namespace.

Reply:
IIRC, Note that while `<cstring>` guarantees that the functions would be in the `std::` namespace, it does not guarantee them being in the global namespace.

Reply:
Thank you for the review.